docker/Dockerfile

```diff
@@ -1,42 +1,20 @@
-ARG RISK=edge
-ARG UBUNTU=xenial
+ARG RISK=stable
+ARG UBUNTU=22.04
```

Comment:
In README.md codename of release is used, probably makes sense to stick to codenames and replace with `jammy` ?

Reply:
I rarely remember which release name is which version. I remember that 22.04 is the latest LTS, and I guess that's just what matters here.